data/json/npcs/classes.json

```diff
@@ -438,5 +438,29 @@
       { "skill": "pistol", "bonus": { "rng": [ 3, 6 ] } },
       { "skill": "rifle", "bonus": { "rng": [ 3, 6 ] } }
     ]
+  },
+  {
+    "type": "npc_class",
+    "id": "NC_Rabbit_Raiser",
```

Comment:
Since this is a unique class, this whole block would work a lot better in the mr_lapin.json file

Reply:
Cool deal that should work :)